Slingshot.Api/Controllers/ARMController.cs

```diff
@@ -395,6 +396,35 @@ private void DeleteResourceGroup(string subscriptionId, string token, string rgN
             return response;
         }
 
+        [Authorize]
+        [HttpPost]
+        public async Task DeploymentNotification(DeployInputs inputs)
+        {
+            string repositoryUrl = HttpUtility.UrlDecode(inputs.repoUrl);
+            string token = GetTokenFromHeader();
+            Repository repo = Repository.CreateRepositoryObj(repositoryUrl, Request.RequestUri.Host, token);
+            var queryStrings = HttpUtility.ParseQueryString(repositoryUrl);
+            if (queryStrings["pr"] != null)
+            {
+                // if deployment is come from a pull request, post a comment back to the pull request.
+                string siteUrl = string.Format(CultureInfo.InvariantCulture, "https://{0}.azurewebsites.net", inputs.parameters["siteName"]["value"]);
+                StringBuilder pullRequestComment = new StringBuilder();
+                pullRequestComment.AppendFormat(CultureInfo.InvariantCulture, "A [website]({0}) has been deployed to Azure from this pull request", siteUrl);
+
+                bool isManualIntegration = true;
+                if (inputs.parameters["isManualIntegration"] != null &&
+                    inputs.parameters["isManualIntegration"]["value"] != null &&
+                    bool.TryParse(inputs.parameters["isManualIntegration"]["value"].ToString(), out isManualIntegration) &&
+                    !isManualIntegration)
+                {
+                    pullRequestComment.Append(" with continuous deployment enabled");
+                }
+
+                pullRequestComment.AppendFormat(". [{0}]({0})", siteUrl);
```

Comment:
They already turn straight URLs into links, so you you don't need URL markdown syntax.